pulp_ansible/app/serializers.py

```diff
@@ -211,6 +217,11 @@ class CollectionRemoteSerializer(RemoteSerializer):
         default=False,
     )
 
+    last_synced_time = serializers.SerializerMethodField()
+
```

Comment:
Can you explain why this field needs to be available for a remote? A remote can be used in multiple concurrent syncs at the same time, so not sure this field would provide much help. Maybe something that lists the last sync it was used in?

Reply:
>Maybe something that lists the last sync it was used in?

I think you're correct, in our current solution, we have the `last_sync_task` which provides more than just the last synced time, but also state and error which we need
```
"last_sync_task": {
        "task_id": "f5e675e7-a429-4616-aea8-d82aae790610",
        "state": "completed",
        "started_at": "2023-01-09T12:05:58.642566Z",
        "finished_at": "2023-01-09T12:06:02.272053Z",
        "error": null
},
```